khal/khalendar/aux.py

```diff
@@ -12,7 +12,7 @@
 logger = log.logger
 
 
-def expand(vevent, href=''):
+def expand(vevent, href='', expand=True):
```

Comment:
Strongly dislike this, I'd prefer if we could move the entire logic into the expand function.